Mono.Cecil/CustomAttribute.cs

```diff
@@ -66,6 +66,7 @@ public interface ICustomAttribute {
 		Collection<CustomAttributeNamedArgument> Properties { get; }
 	}
 
+	[DebuggerDisplay("{AttributeType}")]
```

Comment:
Please add a space before the `(` here and in the other.

Reply:
@jbevain Fixed.  Thanks!